CHANGELOG.md

```diff
@@ -4,6 +4,107 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.13.0 - 2019-02-19
+
+### Added
+
+- When cluster-creation times out waiting for cluster-version
+  completion, the installer now logs the last failing-operator
+  message (if any).
+- The installer now invokes the [cluster-config
+  operator][cluster-config-operator] on the bootstrap machine to
+  generate `config.openshift.io` resources.
+
+### Changed
+
+- The install-config version has been bumped from `v1beta2` to
+  `v1beta3` as `machines` has been split into `controlPlane` and
+  `compute`.  All users will need to update any saved
+  `install-config.yaml` to use the new schema.  As part of this
```

Comment:
> As part of this
  change, multiple compute pools are now supported (previously, only a
  single `worker` pool was supported).

i think that can be a separate bullet.

Reply:
Or atleast separate like the `masterIPs` change. 
Also worth mentioning the `    * The user will be allowed to configure multiple compute machine pools. Every compute machine pool will use the same ignition config.` from https://github.com/openshift/installer/pull/1231

Reply:
> i think that can be a separate bullet.

Separated into sub-list entries with wording based on your Ignition-config suggestion in b6882f2 -> 8f08508.